src/crimson/os/seastore/random_block_manager/nvme_manager.h

```diff
@@ -70,6 +72,8 @@ struct rbm_metadata_header_t {
     denc(v.start_data_area, p);
     denc(v.flag, p);
     denc(v.feature, p);
+    denc(v.blocks_per_segment, p);
```

Comment:
Can we remove the concept of segment from the RandomBlockManager interface and implementations entirely?